apps/common/receipt.py

```diff
@@ -130,17 +128,14 @@ def format_inline_qr(data):
     return Markup(etree.tostring(root).decode("utf-8"))
 
 
-def format_inline_epc_qr(payment):
-    qrfile = make_epc_qrfile(payment, kind="svg", svgclass=None)
+def format_inline_qr(data):
+    qrfile = make_qrfile(data, kind="svg", svgclass=None)
```

Comment:
Maybe you should use:
```
qrfile = make_qrfile(data, kind='svg', svgclass=None, omitsize=True)
```
Since you're replacing the height / width attributes a viewBox in line 125 anyway.
See also <https://segno.readthedocs.io/en/stable/svg-options.html>

Reply:
Brilliant, that allows for a nice bit of code cleanup.  Thanks!